.github/scripts/utils.py/check-jsonschema.py

```diff
@@ -0,0 +1,131 @@
+import json
+import logging
+import os
+import sys
+import argparse
+from typing import Any
+
+from json_source_map import calculate
+from json_source_map.errors import InvalidInputError
+from jsonschema import Draft7Validator
```

Comment:
They were automatically added by the Python language server, I usually don't manage imports manually because the LSP takes care of all that.

In this case the LSP only imported the functions or objects as required by the code.